code/game/objects/structures/lavaland/necropolis_tendril.dm

```diff
@@ -100,3 +88,101 @@ GLOBAL_LIST_INIT(tendrils, list())
 		if(!T.density)
 			T.TerraformTurf(/turf/open/lava/smooth/lava_land_surface, /turf/open/lava/smooth/lava_land_surface, flags = CHANGETURF_INHERIT_AIR) //WS edit, instead of chasms this produces lava instead.
 	qdel(src)
+
+	//these are good for mappers and already see use in some maps.
+
+/obj/structure/spawner/lavaland/goliath
+	mob_types = list(/mob/living/simple_animal/hostile/asteroid/goliath/beast/tendril)
+
+/obj/structure/spawner/lavaland/legion
+	mob_types = list(/mob/living/simple_animal/hostile/asteroid/hivelord/legion/tendril)
+
+/obj/structure/spawner/lavaland/icewatcher
+	mob_types = list(/mob/living/simple_animal/hostile/asteroid/basilisk/watcher/icewing)
+
+/obj/structure/spawner/lavaland/whitesandsbasilisk
+	mob_types = list(/mob/living/simple_animal/hostile/asteroid/basilisk/whitesands)
+
+	//these are ones that we want to see spawning on worlds.
+
+/obj/structure/spawner/lavaland/low_threat //this is the most common one, it shouldn't be a huge issue for most players.
+	mob_types = list(
+		/mob/living/simple_animal/hostile/asteroid/goliath/beast/tendril = 25,
+		/mob/living/simple_animal/hostile/asteroid/hivelord/legion/tendril = 25,
+		/mob/living/simple_animal/hostile/asteroid/basilisk/watcher/tendril = 25,
+		/mob/living/simple_animal/hostile/asteroid/basilisk/watcher/icewing = 5,
+		/mob/living/simple_animal/hostile/asteroid/brimdemon = 20
+	)
+	max_mobs = 5
+	spawn_time = 300
+
+/obj/structure/spawner/lavaland/medium_threat //this is less common. It starts getting dangerous here.
+	mob_types = list(
+		/mob/living/simple_animal/hostile/asteroid/goliath/beast/tendril = 25,
+		/mob/living/simple_animal/hostile/asteroid/hivelord/legion/tendril = 25,
+		/mob/living/simple_animal/hostile/asteroid/basilisk/watcher/tendril = 25,
+		/mob/living/simple_animal/hostile/asteroid/basilisk/watcher/icewing = 5,
+		/mob/living/simple_animal/hostile/asteroid/brimdemon = 20
+	)
+	max_mobs = 8
```

Comment:
This feels like too much. I would reduce low threat to four and this to six, probably.